cmd/frpc/sub/root.go

```diff
@@ -37,17 +37,19 @@ import (
 )
 
 var (
-	cfgFile          string
-	cfgDir           string
-	showVersion      bool
-	strictConfigMode bool
+	cfgFile                string
+	cfgDir                 string
+	showVersion            bool
+	strictConfigMode       bool
+	allowUnsafeTokenSource bool
 )
 
 func init() {
 	rootCmd.PersistentFlags().StringVarP(&cfgFile, "config", "c", "./frpc.ini", "config file of frpc")
 	rootCmd.PersistentFlags().StringVarP(&cfgDir, "config_dir", "", "", "config directory, run one frpc service for each file in config directory")
 	rootCmd.PersistentFlags().BoolVarP(&showVersion, "version", "v", false, "version of frpc")
 	rootCmd.PersistentFlags().BoolVarP(&strictConfigMode, "strict_config", "", true, "strict config parsing mode, unknown fields will cause an errors")
+	rootCmd.PersistentFlags().BoolVarP(&allowUnsafeTokenSource, "allow_unsafe_token_source", "", false, "enables usage of exec-based token source")
```

Comment:
Since we may add other unsafe capabilities in the future, I suggest generalizing the flag to `--allow-unsafe` and making it accept a list/array. For now we would only support "TokenSourceExec".